bin/pod/plotting_POD.R

```diff
@@ -0,0 +1,166 @@
+# create lattice dotplots to illustrate where missmatches or false matches are along chromosome
```

Comment:
This script is not activated in the pipeline here - just copied from the container to make them easier to view, and eventually incorporate as a regular binary in the pipeline